extras/docker/centos/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM registry.centos.org/centos/centos:latest
+MAINTAINER Heketi Developers <heketi-devel@gluster.org>
+
+LABEL version="0.8"
```

Comment:
What does this version number mean? What does it correspond to?

Reply:
I think it is the version of this `Dockerfile`, no idea where it is used. The one in `../fromsource/` uses version 1.3.1 or such.

Reply:
Yeah, I noticed that as well. I'm starting to suspect these lables are generally ignored.